src/utils.rs

```diff
@@ -81,9 +81,9 @@ pub fn snippet_block<'a>(cx: &Context, span: Span, default: &'a str) -> Cow<'a,
 /// Trim indentation from a multiline string
 /// with possibility of ignoring the first line
 pub fn trim_multiline(s: Cow<str>, ignore_first: bool) -> Cow<str> {
-    let s = trim_multiline_inner(s, ignore_first, ' ');
-    let s = trim_multiline_inner(s, ignore_first, '\t');
-    trim_multiline_inner(s, ignore_first, ' ')
+    let s_space = trim_multiline_inner(s, ignore_first, ' ');
```

Comment:
Well, we should in any case not add/activate lints that lead to less ideal code.